public/js/webcam.js

```diff
@@ -71,16 +80,29 @@ function postTracks(dataURL) {
             location.href = "/tracks";
         })
         .catch((error) => {
-            alert(`Error ${error.response.status}: ${error.response.statusText}`);
+            hide([loadingDiv]);
+            show([webcam, errorAlert]);
+            $("#header").hide();
+            $("#errorAlert").append("<span>"+"Sorry, we encountered an error. " + "</span>");
+
+            if(error.response) {
+                $("#errorAlert").append("<span>" + error.response.data.errorMsg +"</span>");
```

Comment:
Although the effect would be small, should we append it the message before showing the erroralert?

Reply:
Same comment above for lines 36, 37